src/design/type-tokens/type-tokens.mdx

```diff
@@ -20,216 +20,216 @@ consistent and scalable design system.
 
 ## <span className="h2">Font Tokens</span>
 
-### <span className="h3">Display</span>
+## <span className="h3">Heading</span>
 
 <span className="p">
-  This styling utilizes Distinct Display in Upper Case. Typically these are used within{' '}
-  <strong>Banners, Cards & Interstitials</strong>
-  for brand/Auction titling. There are five sizes in this scale:
+  This styling utilizes Distinct Display in title case. Typically used within <strong>layouts</strong> for hierarchy
+  flow; There are four sizes in this scale:
 </span>
+
 <h4 className="h4">
-  Display 0: <span className="token">`$display0`</span>
+  Heading 1: <span className="token">`$heading1`</span>
 </h4>
 
 - **Default**:
-  <span className="property">font-size:</span> 3.25rem;
-  <span className="property">line-height:</span> 1.25;
-  <span className="property">font-weight:</span> 400;
-  <span className="property">text-transform:</span> uppercase;
-- **BP1**: <span className="property">font-size:</span> 2.59rem;
-- **BP4**: <span className="property">font-size:</span> 4.06rem;
+  <span className="property">font-size:</span> 1.75rem;
+  <span className="property">line-height: </span>2rem;
+  <span className="property">font-weight:</span>400;
+  <span className="property">text-transform:</span> capitalize;
+- **BP1**: <span className="property">font-size:</span> 1.5rem; <span className="property">line-height:</span> 1.75rem;
+- **BP4**: <span className="property">font-size:</span> 2rem; <span className="property">line-height:</span> 2.25rem;
 
 <details>
   <summary>Example</summary>
+</details>
 
 ```scss
-@include text($display0);
+@include text($heading1);
 ```
 
-  <span className="example example--d0">example</span>
-</details>
```

Comment:
I think this is the part that broke the example.

Reply:
this is fixed now @davidicus